libtiledbsoma/src/soma/soma_array.h

```diff
@@ -633,7 +633,7 @@ class SOMAArray : public SOMAObject {
      * @return Throws if the requested shape exceeds the array's create-time
      * maxshape. Throws if the array does not have current-domain support.
      */
-    void resize_soma_joinid_if_dim(const std::vector<int64_t>& newshape);
+    void maybe_resize_sjid(const std::vector<int64_t>& newshape);
```

Comment:
Can you put in docs `soma_joinid (sjid)`? The abbreviation wasn't obvious to me at first.

Reply:
tbh i'll just `sjid` -> `soma_joinid` ... it's overly cryptic ...